### DISCUSSION TITLE
Publish in a readable screenplay format

### DISCUSSION THREAD

**Discussion by russellaugust** [chuangcaleb/obsidian-fountain-editor#24] [Ideas]
I'd love if there was a way to Publish it in the screenplay formatting. Not sure if Publish lets you just include more css in this way or not.

**Comment by chuangcaleb** [comment]
I'd love that too. Do you mean Obsidian Publish? If it is, I'm not too familiar + I don't have personal incentive since I don't use it myself. If you do and don't mind getting it working within this plugin, feel free to contribute a pull request! :D

**Reply by russellaugust** [reply]
I'll see what I can do. I'm hoping it's as easy as modifying the publish.css but I need to familiarize myself with the CSS style names in Obsidian.
